### PR TITLE
Format third-level category menu

### PR DESCRIPTION
**Pull request by ourmaninamsterdam**
@asmega @darokel

### CODE REVIEW

File: app/views/corporate/_content_items.html.erb [hunk 1]
```diff
@@ -1,15 +1,9 @@
 <div class="l-inpage-nav">
```
Comment (ourmaninamsterdam) [inline app/views/corporate/_content_items.html.erb]: I noticed there are two partials doing the same thing, is this correct?
* app/views/corporate/_content_items.html.erb
* app/views/corporate_categories/_content_items.html.erb

**Comment by ourmaninamsterdam** [comment]
Anyone?? @moneyadviceservice/team-bee

File: app/views/corporate/_content_items.html.erb (hunk 1 above)
Comment (darokel) [inline app/views/corporate/_content_items.html.erb]: Yeah. It's not ideal but it's due to the way things have already been implemented. One is for rendering corporate category items and the other is for `/en/corporate` items.

**Comment by darokel** [comment]
:+1:

**Comment by asmega** [comment]
:+1:

**Pull request merged**